pkg/pipelineascode/pipelineascode.go

```diff
@@ -136,6 +136,12 @@ func (p *PacRun) startPR(ctx context.Context, match matcher.Match) (*tektonv1.Pi
 	// Add labels and annotations to pipelinerun
 	kubeinteraction.AddLabelsAndAnnotations(p.event, match.PipelineRun, match.Repo, p.vcx.GetConfig())
 
+	// Add annotations to PipelineRuns to integrate with Tekton Results
+	err := kubeinteraction.AddResultsAnnotation(p.event, match.PipelineRun)
```

Comment:
what about you move that function to add this to AddLabelsAndAnnotations function instead ? so we can have all annotation in one place. it's small enough to don't have to be it's own,..

Reply:
Sure, that sounds good to me. Although, then I'll have to return an error with AddLabelsAndAnnotations function since this marshal function [here](https://github.com/openshift-pipelines/pipelines-as-code/pull/1245/files#diff-7c7d6fa80091c2239e1f4ccd8655302b1461cdccbed787ce97519a55536eb57fR38) returns an error.